astromodels/core/polarization.py

```diff
@@ -1,10 +1,102 @@
 __author__ = 'giacomov'
 
 from astromodels.core.tree import Node
+from astromodels.core.parameter import Parameter
 
 
 class Polarization(Node):
 
-    def __init__(self):
+    def __init__(self, type='linear'):
+
+        assert type in ['linear', 'stokes'], 'polarization must be linear or stokes'
```

Comment:
Please do not use `type` as variable, as it is a python protected keyword.

Reply:
Doh! Speed coding.